src/game.py

```diff
@@ -0,0 +1,249 @@
+# make_sudoku.py
+# author: David Bau
+# Copyright (c) 2006 David Bau.  All rights reserved.
+# URL: http://davidbau.com/downloads/sudoku.py
+# Modified by Daniel Jauregui for study purposes
+
+import random
+
+
+class Game(object):
+
+    def __init__(self, level):
+        self.level = level
+
+    def generate_game(self):
+        """Call to make_puzzle with a empty board to generate game
+        with level defined and it will return a array e.g.: Array: [4, 9, 3, None, 5, ...., 8, 8, None, None, 1]
+        Where the position are from 0 to 80
+
+        Keyword arguments:
```

Comment:
If there is no arguments on the method's signature we can remove this and only leave the return row